.github/workflows/BUILD_AND_PACKAGE_REUSABLE.yml

```diff
@@ -308,15 +308,12 @@ jobs:
             invoke test -m ${{env.testplatform}} -n physical -n arm
           fi
 
-      - uses: dorny/test-reporter@v1
+      - name: Upload test report
+        uses: actions/upload-artifact@v3
         if: success() || failure()
         with:
-          name: Test Results ${{env.platform}} ${{env.osnick}} ${{env.arch}}
+          name: test-report-${{env.platform}}-${{env.osnick}}-${{env.arch}}
```

Comment:
Was there a reason for the name change?

Reply:
For now this name represent a name of artifact and not the name of actual test result. The name of Test result will remain the same and specified here

https://github.com/redis-stack/redis-stack/pull/516/files#diff-5c0f6ce4ba8bd452137d2d62d5be9aff64ac2055cb4183df15e46926e999faebR21

Reply:
And the suite and test lists?